domain-ee/ee-ep-merge-app/src/python_src/api.py

```diff
@@ -192,14 +162,7 @@ async def get_merge_jobs(state: Annotated[list[JobState], Query()] = JobState.in
 async def sqlalchemy_exception_handler(request: Request, err: SQLAlchemyError):
     msg = str(err).replace('\n', ' ')
     logging.error(f"event=requestFailed method={request.method} url={request.url} resource={'Database'} error={msg}")
-    return JSONResponse(
-        status_code=500,
-        content=jsonable_encoder({
-            "method": request.method,
-            "url": str(request.url),
-            "errors": [CONNECT_TO_DATABASE_FAILURE]
-        })
-    )
+    return JSONResponse(status_code=500, content=jsonable_encoder({"method": request.method, "url": str(request.url), "errors": [CONNECT_TO_DATABASE_FAILURE]}))
```

Comment:
This was perhaps the only instance where collapsing multiple lines into a single line felt a little heavy-handed, and yeah part of it might be that the single line is pretty much maxed out at 160 chars, but I actually think it's more due to the fact that it was multiple levels of indentation which you can't see in a single line.

There's no option to have Black preserve multiple levels of indentation anyway, so that's moot.

Reply:
160 does feel a little long. I'm on an ultrawide monitor and 120-140 seems to be the sweet spot with 2 files open side-by-side in VSCode.